lib/matplotlib/tests/test_axes.py

```diff
@@ -3600,6 +3600,16 @@ def test_errorbar():
     ax.set_title("Simplest errorbars, 0.2 in x, 0.4 in y")
 
 
+@image_comparison(['errorbar_polar_caps'], extensions=['png'])
+def test_errorbar_polar_caps():
+    fig = plt.figure()
+    ax = plt.subplot(111, projection='polar')
+    theta = np.arange(0, 2*np.pi, np.pi / 4)
+    r = theta / np.pi / 2 + 0.5
+    ax.errorbar(theta, r, xerr=0.15, yerr=0.1, fmt="o")
```

Comment:
Maybe add additional datasets with:
- individual error bar sizes per data point
- asymmetric +/- errors
- a quite large theta-error (~pi/2 - this should still be curvy enough with 12 interpolation steps. I think it does not make sense to add a test for the case that the interpolation is not good anymore, because we may change this in the future - e.g. adapt the interpolation steps so that a step is never larger than a certain angle).

Can be 1 to few point per variant.

Reply:
I have added more test cases, apart from the even errorbars defined by constant x/y err I mixed asymmetric and individual error bar sizes per datapoint. Additionally I have added long errorbars, and overlapping errorbars - these are not nice or useful, but on other hand it demonstrates that nothing goes particularly wrong.

With regards to _interpolation_steps, I changed the number to 2. It seems that it does work the same for all positive nubers greater than 1. I am not sure if this is desired behavior, but I didn't any deep investigation.

Reply:
Thanks! Could you join them into only one? We try quite hard to limit the amount of test images, because they blow up the size of the repo. The four tests would be an additional 200kb, which sounds not much, but we have thousands of tests, so it adds up.

Suggestion:
- outer most radius: one overlapping bar
- next inner radius: one large bar
- further in: one half for even bars, the other half for asymmetric bars.

If you plot them all into one Axes, automatic color cycling should give them different colors so that the different parts of the test are still easy to distinguish.

*Note*: I cannot comment on `_interpolation_steps` as that's not my topic, but emirically, it seems to work.

Reply:
The fact that any _interpolation_steps > 1 works the same for fixed-radius arcs is intended, see https://github.com/matplotlib/matplotlib/pull/14852 (first paragraph of the first message).  Probably it could be better documented, though.

Reply:
I have added the mix of errorbar options in single figure. But test failed for overlapping errorbars (see [this report](https://github.com/matplotlib/matplotlib/runs/7938210126?check_suite_focus=true) ). Therefore, for now, I have removed the overlapping case from the figure since it is not a real use case and its testing seems unstable.